include/pmacc/particles/ParticlesBase.kernel

```diff
@@ -656,7 +656,7 @@ struct KernelShiftParticles
 
                 if ( tmpFrame.isValid() )
                 {
-                    uint32_t particlesInFrame = pb.getSuperCell( relativeCtx[ idx ] ).getSizeLastFrame( );
+                    int const particlesInFrame = pb.getSuperCell( relativeCtx[ idx ] ).getSizeLastFrame( );
```

Comment:
```suggestion
                    int32_t const particlesInFrame = pb.getSuperCell( relativeCtx[ idx ] ).getSizeLastFrame( );
```

Reply:
Then also change the type of `newParticleInFrame`, since it is now `int` and not `int32_t`?

Reply:
I would say so, looks sloppy.